python/aqora/iceberg.py

```
@@ -0,0 +1,169 @@
+        self._executable = executable or shutil.which("aqora") or "aqora"
+        self._token: str | None = None
+        self._expires_at: float = 0.0
+        self._lock = threading.Lock()
+
+    def _fetch_token(self) -> str:
+        cmd: list[str] = [self._executable]
+        if self._aqora_url is not None:
+            cmd.extend(["--url", self._aqora_url])
+        cmd.extend(["auth", "token"])
+        try:
+            result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)